tests/test_meta.py

```diff
@@ -0,0 +1,13 @@
+import pathlib
```

Comment:
please add the https://github.com/Open-MSS/MSS/blob/develop/new_file_template.py

Reply:
done